src/DssSpell.sol

```diff
@@ -88,14 +108,17 @@ contract DssSpell {
         expiration = now + 30 days;
     }
 
-    function schedule() public {
+    function schedule() external {
         require(now <= expiration, "This contract has expired");
         require(eta == 0, "This spell has already been scheduled");
         eta = now + DSPauseAbstract(pause).delay();
         pause.plot(action, tag, sig, eta);
+
+        DSAuthAbstract(SAI_MOM).setAuthority(address(0));
```

Comment:
Should we have a human-readable comment above these? I don't see them mentioned in the governance document so we might want to mention why we're doing this.

Reply:
Comment added, please feel free to propose something better if it doesn't look very good.